format_parser.gemspec

```diff
@@ -31,16 +32,16 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'ks', '~> 0.0'
-  spec.add_dependency 'exifr', '~> 1', '>= 1.3.8'
```

Comment:
Do we need to change the version requirements for **all** dependencies?

Reply:
Need? No. But ten birds, one stone - why would we not?

Reply:
Because we're narrowing down client support in cases where that might not be necessary.